communication/moderators.md

```diff
@@ -82,11 +82,11 @@ Administrators seats: 4
 
 | Name          | Kubernetes Discuss ID | Region   | Timezone                                                |
 | ------------- | --------------------- | -------- | ------------------------------------------------------- |
-| Jorge Castro  | @castrojo             | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
```

Comment:
castrojo is still an admin, it's alejandrox1 that has been removed. I should have been more clear as there are two jorges^^;;;;;

Reply:
Updated and sorted alphabetically.